docs/source/template_options.rst

```diff
@@ -0,0 +1,266 @@
+1. Simple or custom installation
+--------------------------------
+
+   +------------+----------------------------------------------------------------------------+
+   | Question   | Would you like to use simple (default tooling) or customized installation? |
+   +------------+----------------------------------------------------------------------------+
+   | Options    | **✱ customized**, simple                                                   |
+   +------------+----------------------------------------------------------------------------+
+
+.. Something I'm struggling with is how to demarkate the default values that a simple install uses.
+
+Choosing "simple installation" will use the default values listed at the top of each of the following sections (labeled `Default` or marked with a ✱).
+
+
+2. Name of your project
+-----------------------
+
+   +------------+-----------------------------------------------+
+   | Question   | What is the name of your project?             |
+   +------------+-----------------------------------------------+
+   | Default    | `my_project`                                  |
+   +------------+-----------------------------------------------+
+
+The name of your project.
+
+Must start with a lowercase letter, followed by one or more of the following (a-z, 0-9, _, -).
+
+If you distribute your code via PyPI, this is the name that will be used. This will allow users to install like so: ``pip install <project_name>``. 
+
+Note that this will be used to connect to your project on github, as in ``https://github.com/{{project_organization}}/{{project_name}}``.
+
+
+3. Python package name
+-----------------------
+
+   +------------+-----------------------------------------------+
+   | Question   | What is the name of your Python package?      |
+   +------------+-----------------------------------------------+
+   | Default    | `example_package`                             |
+   +------------+-----------------------------------------------+
+
+The name of your top level package. 
+
+Must start with a lowercase letter, followed by one or more of the following (a-z, 0-9, _).
+
+Specifies the location of the source code, as in::
+
+    project_name/
+    ├─ src/
+    │  ├─ package_name/
+    │  │  ├─ __init__.py
+    │  │  ├─ module.py
+    │  │  ├─ ...
+
+
+
+4. Github organization name
+----------------------------
+
+   +------------+--------------------------------------------------------+
+   | Question   | What github organization will your project live under? |
+   +------------+--------------------------------------------------------+
+   | Default    | `my-organization`                                      |
+   +------------+--------------------------------------------------------+
+
+This will be:
+
+    * The name of the GitHub organization that will host your project, or
```

Comment:
There are many benefits to using a github organization that astro folks tend to not be aware of, and we should probably be advocating this approach more.

But that's a little outside the scope of this PR.